### DISCUSSION TITLE
Allow custom collection name

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#2974] [Feature Requests]
Currently name of the collection is from the DB collection name. Can we create a custom name for that used only in the UI?

**Comment by Michael-Nussbaumer** [comment]
You can add translations to the collection:

![Screenshot_43](https://user-images.githubusercontent.com/15796397/98946575-a1e42300-2526-11eb-8499-431c2cdd275a.png)

This should solve your problem :)
